conf/grub.cfg

```diff
@@ -3,3 +3,5 @@
 #   set_global eve_flavor kvm
 # to force booting in Xen mode, uncomment:
 #   set_global eve_flavor xen
+# set_global dom0_cmdline eve_inventory_server=http://10.129.17.151:8888
+set_global dom0_cmdline "$dom0_cmdline eve_inventory_server=http://192.168.1.55:8888"
```

Comment:
surely last line was included by accident?

Reply:
This is an example in this draft. Will be removed.